backend/typescript/services/implementations/EntityServicePg.ts

```diff
@@ -51,13 +62,22 @@ class EntityService implements IEntityService {
 
   async createEntity(entity: EntityRequestDTO): Promise<EntityResponseDTO> {
     let newEntity: PgEntity | null;
+    const fileName = entity.filePath ? uuidv4() : "";
     try {
+      if (entity.filePath) {
+        this.storageService.createFile(
```

Comment:
Same thing as the mongo service, please add await before all calls to `storageService` methods :)